.github/workflows/ci.yml

```diff
@@ -73,8 +73,6 @@ jobs:
           go-version: '^1.15.0'
 
       - run: go test -gcflags=all=-d=checkptr -v ./... -tags admin
-      - run: go test -gcflags=all=-d=checkptr -v ./internal -tags admin
-        working-directory: test
```

Comment:
why this?

Reply:
The only file in `test/internal` was a schemaversion test file, which I moved into the code library.
It was only ever in the test directory because it needed a manifest file to get the OS version, but with the manifest being moved out into `testutil`, it can be imported into unit tests in the main code base without any issues